<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QToolButton" name="emojiButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QToolButton{
background-color:white;
border-color:white;
border-width:0px;
padding:0 0 0 0;
border:solid;
}

QToolButton:pressed { 
	background-color:green;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="rsc.qrc">
       <normaloff>:/image/个人中心.png</normaloff>:/image/个人中心.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="pictureButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QToolButton{
background-color:white;
border-color:white;
border-width:0px;
padding:0 0 0 0;
border:solid;
}

QToolButton:pressed { 
	background-color:green;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="rsc.qrc">
       <normaloff>:/image/图片.png</normaloff>:/image/图片.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="voiceButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QToolButton{
background-color:white;
border-color:white;
border-width:0px;
padding:0 0 0 0;
border:solid;
}

QToolButton:pressed { 
	background-color:green;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="rsc.qrc">
       <normaloff>:/image/语音.png</normaloff>:/image/语音.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="CBaseChatEdit" name="textEdit"/>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="submit">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="rsc.qrc">
       <normaloff>:/image/发送.png</normaloff>:/image/发送.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CBaseChatEdit</class>
   <extends>QTextEdit</extends>
   <header>cbasechatedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
